stdlib/public/runtime/Errors.cpp

```diff
@@ -72,7 +72,6 @@ static bool getSymbolNameAddr(llvm::StringRef libraryName, Dl_info dlinfo,
   // If the address is unavailable, just use <unavailable> as the symbol
```

Comment:
nit-pick: Since you remove the "<unavailable>" string below, this comment here is no longer accurate. Could you update this comment as well?